src/MongoDB/Manager.c

```diff
@@ -301,7 +301,70 @@ static PHP_METHOD(Manager, executeCommand)
 
 	intern = Z_MANAGER_OBJ_P(getThis());
 
-	phongo_execute_command(intern->client, db, command, options, -1, return_value, return_value_used TSRMLS_CC);
+	phongo_execute_command(intern->client, PHONGO_COMMAND_RAW, db, command, options, -1, return_value, return_value_used TSRMLS_CC);
+} /* }}} */
+
+/* {{{ proto MongoDB\Driver\Cursor MongoDB\Driver\Manager::executeReadCommand(string $db, MongoDB\Driver\Command $command[, array $options = null])
+   Execute a ReadCommand */
+static PHP_METHOD(Manager, executeReadCommand)
+{
+	php_phongo_manager_t     *intern;
+	char                     *db;
+	phongo_zpp_char_len       db_len;
+	zval                     *command;
+	zval                     *options = NULL;
+	DECLARE_RETURN_VALUE_USED
+	SUPPRESS_UNUSED_WARNING(return_value_ptr)
+
+	if (zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "sO|z!", &db, &db_len, &command, php_phongo_command_ce, &options) == FAILURE) {
```

Comment:
Should you use `a!` for `$options` here? There is no BC concern, so users should always pass an array type.

Reply:
Done